internal/fs/fs.go

```diff
@@ -280,6 +287,19 @@ func copyFile(src, dst string) (err error) {
 	return
 }
 
+// copySymlink will resolve the src symlink and create a new symlink in dst.
+// If src is a relative symlink, dst will also be a relative symlink.
+func copySymlink(src, dst string) error {
+	resolved, err := os.Readlink(src)
+	if err != nil {
+		return errors.Wrap(err, "failed to resolve symlink")
+	}
+
+	err = os.Symlink(resolved, dst)
+
+	return errors.Wrapf(err, "failed to create symlink %s to %s", src, resolved)
```

Comment:
This is awkwardly halfway between a terse form and the standard form. Either combine this and the preceding line into one (`errors.Wrapf(os.Symlink(resolved, dst), ...)`), or do the more standard `if err != nil` check.